templates/observation.html

```diff
@@ -313,18 +313,20 @@ <h4 class="text-center">Students</h4>
     </div>
     <div class="row">
       <div class="col-12 d-flex justify-content-between">
-        {% if draft_observation and last_observation_url %}
-          <div>
+        <div>
+          <button type="submit" id="re-setup-btn" class="btn btn-primary btn-lg">Go back to setup</button>
```

Comment:
This label in not verbose enough. It suggest that this is usuall back. Maybe: `Reconfigurate observation template`? This also is not perfect. Do you have any other suggestion?